ansible_rulebook/vault.py

```
@@ -59,34 +61,36 @@ def __init__(
-        child.sendline(vault_text)
-        child.sendcontrol("D")
-        i = child.expect(["Decryption successful", "ERROR"])
-        if i == 0:
-            child.readline()
-            decrypted_text = "".join(line.decode() for line in child)
-            return decrypted_text.strip()
-        else:
-            error_msg = child.readline()
-            raise VaultDecryptException(error_msg.decode())
+
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)